OmiseSDK/Capability.swift

```diff
@@ -69,8 +69,15 @@ extension Capability {
             case truemoney
             case points(PaymentInformation.Points)
             case eContext
+            case fpx
             case unknownSource(String, configurations: [String: Any])
         }
+
+        public struct Bank: Codable, Equatable {
+            public let name: String
+            public let code: String
+            public let active: Bool
+        }
```

Comment:
Can you rename `active` to `isActive`? In swift it's very common to prefix Booleans with `is`. 
But then you also have to add `CodingKeys` like this:
```suggestion
        public struct Bank: Codable, Equatable {
            enum CodingKeys: String, CodingKey {
                case name, code
                case isActive = "active"
            }
            
            public let name: String
            public let code: String
            public let isActive: Bool
        }
```